data_safe_haven/external/api/graph_api.py

```
@@ -682,7 +684,7 @@ def grant_application_role_permissions(
                 "resourceId": microsoft_graph_sp["id"],
                 "appRoleId": app_role_id,
             }
-            response = self.http_post(
+            self.http_post(
```

We don't need to check the response here to check the operation was successful?

---

It should raise an exception if it fails.